### PR TITLE
232 app boilerplate

### PR DESCRIPTION
**Pull request by katjam**
Trello card: 
https://trello.com/c/xnffqen9/232-vvv-ee-boilerplate

## Description

- Basic init Browser doc program
- Code format guidelines

@neontribe/sea-map

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,10 @@
 - [nvm for macOS & Linux](https://github.com/nvm-sh/nvm) or [nvm for Windows](https://github.com/coreybutler/nvm-windows)
 - [Parcel](https:://parceljs.org)
```
Comment (Nikomus) [inline README.md]:
```suggestion
- [Parcel](https://parceljs.org)
```
Missed this in the last PR, sorry!

File: README.md [hunk 2]
```diff
@@ -11,6 +11,10 @@
...
 
+### Formatting
+We recommend integrating `elm-format@0.8.3` into your code editor, but if you don't...
+- Please `npm run format` to format `.elm` files in `src` before committing code.
+
 ### Develop
 - `npm start` for a hot-reload dev server
```
Comment (Nikomus) [inline README.md]: Could maybe add that parcel dev server is http://localhost:1234/

File: src/Main.elm [hunk 3]
```diff
@@ -0,0 +1,46 @@
+module Main exposing (main)
+
+import Browser
+import Html
+
+
+main : Program () Model Msg
+main =
+    Browser.document
```
Comment (Nikomus) [inline src/Main.elm:9]: Guessing Browser.application is too much for now?

Comment (katjam) [reply src/Main.elm:9]: Yeah - I thought this was a good first step. We might end up embedding in their wordpress - so I toyed with using element even... but this is simpler as a standalone for now.

File: README.md (hunk 2 above)
Comment (katjam) [reply README.md]: Good idea.

Comment (katjam) [reply README.md]: Done @Nikomus review again at your leisure!

**Review by Nikomus — APPROVED** [review]
A good start! 🚕

**Pull request merged**